app/models/relay.rb

```diff
@@ -68,7 +68,7 @@ def unfollow_activity(activity_id)
   end
 
   def some_local_account
-    @some_local_account ||= Account.local.find_by(suspended: false)
+    @some_local_account ||= Account.find_local(Setting.site_contact_username.gsub(/\A@/, '')) || Account.local.find_by(suspended: false)
```

Comment:
Perhaps create `Account.representative` method?

Reply:
OK.
Sorry for many force-pushes, all done.